### DISCUSSION TITLE
Feature Request: Allow Author To Order Files In PR Review

### DISCUSSION THREAD

**Discussion by samuel-wilder** [community/community#116659] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

Pull requests would be easier to review if there was an option for the author to set the order of each file to review with unordered files sorted to the bottom. Right now, they are all jumbled in there together in an order that is not useful. Given enough use of this feature, GitHub could use heuristics or machine learning to auto-sort files in a PR as a future enhancement.

**Comment by kjw-enf** [comment]
Similar but different (perhaps easier to implement) is a different "smart" sort order. For example, put .h files before the same-named .cc files: i.e. sort order of
a.h
a.cc
b.h
b.cc

For example, put tests after. i.e. sort order of
w.py
w_test.py
x.py
x_test.py
(arguably, alphabetical already covers this specific use case, but may affect other languages and naming schemes differently)
